lib/ex_auction/strategies/english.ex

```diff
@@ -30,12 +30,14 @@ defmodule ExAuction.Strategies.English do
   end
 
   @impl true
-  def winning_bid(%State{bids: bids}) do
+  def winning_bid(%State{bids: [winning_bid | _others]}) do
     # Since all bids in the state are already sorted in descending order, we are safe to return the highest bid
-    [winning_bid | _others] = bids
-
     {:ok, winning_bid}
   end
 
+  def winning_bid(%State{bids: _empty_bids_list}) do
```

Comment:
this pattern is already matched on L42. Returned values should be consolidated, I don't see L42 ever matched.